urbanairship-preference-center-react-native/ios/AirshipPreferenceCenterModule.m

```diff
@@ -45,11 +45,18 @@ @implementation AirshipPreferenceCenterModule
                     NSArray* items = section.items;
                     if (items) {
                         NSMutableArray *itemArray = [NSMutableArray array];
-                        for (id<UAPreferenceItem> item in items) {
+                        for (id item in items) {
+                            
+                            id<UAPreferenceItem> preferenceItem = item;
```

Comment:
Can you include this in the for ?
Like ```for (id<UAPreferenceItem> item in items) {```

Reply:
I want to do that but it will not be possible to cast item to `UAPreferenceChannelSubscriptionItem`  then

Reply:
ah ok, let's keep it like this then